pyopia/exampledata.py

```diff
@@ -4,6 +4,33 @@
 import gdown
 
 
+def get_classifier_database_from_pysilcam_blob(download_directory='./'):
```

Comment:
I think it would be useful if these download functions always returned the download_direction. Then you do not have to duplicate the location in the case where you use the default value.